src/main/java/games/strategy/engine/data/TechnologyFrontierList.java

```diff
@@ -18,12 +18,6 @@ public void addTechnologyFrontier(final TechnologyFrontier tf) {
     m_technologyFrontiers.add(tf);
   }
 
-  public void addTechnologyFrontier(final List<TechnologyFrontier> tfs) {
```

Comment:
I couldn't find any reflection references using similar search parameters as above.

There are direct calls to the other `TechnologyFrontierList#addTechnologyFrontier()` overload.

Searching through the history, this overload was added in e4f8837.  There have been no commits since that time that were calling this overload.  So possibly a YAGNI.